Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -973,20 +973,20 @@ static inline Ref<CSSPrimitiveValue> adjustLengthForZoom(const Length& length, c
     return adjust == ComputedStyleExtractor::AdjustPixelValuesForComputedStyle::Yes ? zoomAdjustedPixelValue(length.value(), style) : CSSValuePool::singleton().createValue(length);
 }
 
-Ref<CSSValue> ComputedStyleExtractor::valueForShadow(const ShadowData* shadow, CSSPropertyID propertyID, const RenderStyle& style, AdjustPixelValuesForComputedStyle adjust)
+Ref<CSSValue> ComputedStyleExtractor::valueForShadow(const ShadowData* shadow, CSSPropertyID propertyID, const RenderStyle& renderStyle, AdjustPixelValuesForComputedStyle adjust)
```

Comment:
I would keep `style`, at least in this file I don't think we use `renderStyle` much compared to just `style`.